server/utils.lua

```diff
@@ -109,6 +109,37 @@ function setHandlingFromTable ( vehicle, tab, exe )
     return true
 end
 
+function outputHandlings(player2,cmd,target)
```

Comment:
sorry, i know there's a lot to change here. the formatting requires spaces between args/commas, see the code for `onRemoteLockRequest` below

a for loop would be `for x, y in iter(this) do`